distribution/docs/src/main/resources/draft-documentation.adoc

```diff
@@ -0,0 +1,39 @@
+= DRAFT Distributed Data Framework Documentation
+include::${project.build.directory}/doc-contents/_contents/config.adoc[]
+
+== ${ddf-registry} Application Reference
```

Comment:
I got feedback from several people that "Managing {app-name} was confusing. These sections are getting smaller and more of a "reference" section with installation and configuration details. I'm happy to change it if we don't like it.